codemp/server/sv_main.cpp

```diff
@@ -38,10 +38,12 @@ cvar_t	*sv_timeout;			// seconds without any message
 cvar_t	*sv_zombietime;			// seconds to sink messages after disconnect
 cvar_t	*sv_rconPassword;		// password for remote server commands
 cvar_t	*sv_privatePassword;	// password for the privateClient slots
+cvar_t	*sv_allowDownload;
+cvar_t	*sv_httpdownloads;
+cvar_t	*sv_httpserverport;
```

Comment:
just a nitpick; should we use camel-caseing for new cvars?

Reply:
Camel cased the new cvars